README.md

```diff
@@ -71,11 +71,29 @@ If you want to update the initial Keycloak configuration, you need to run:
     docker-compose run -d --name keycloak-export keycloak1 --server-config standalone-ha.xml -Dkeycloak.migration.action=export -Dkeycloak.migration.provider=singleFile -Dkeycloak.migration.file=/tmp/initial-import.json -Dkeycloak.migration.usersExportStrategy=REALM_FILE
 
 It takes around a minute before the export is done. Look at the logs:
-    
+
     docker logs keycloak-export | grep "Export finished successfully"
-    
-After that:    
-    
+
+After that:
+
     docker cp keycloak-export:/tmp/initial-import.json test/keycloak/initial-import.json
     docker stop keycloak-export
-    docker rm keycloak-export
\ No newline at end of file
+    docker rm keycloak-export
+
+## Building Prometheus event exporter
+
+__Note:__ This can be deprecated once the aerogear team stars publishing the jars into a Maven repository
+
+Pick a revision:
```

Comment:
The version number is in the metrics/Dockerfile. We should either remove it from there or the instructions should say to change the version number in there. I find it a little bit confusing to have the version number in two places.

Should we make it clear that we need to commit the new jar, the new version (if we go for the Docker file option) and delete the old jar?

Reply:
The revision number is a default if you don't pass an `--build-arg`